docs/topics/whatsnew21.md

```diff
@@ -77,11 +77,11 @@ sealed interface Animal {
 
 fun feedAnimal(animal: Animal) {
     when (animal) {
-        // Branch with only the primary condition. Returns `feedDog()` when `Animal` is `Dog`
+        // Branch with only the primary condition. Calls `feedDog()` when `animal` is `Dog`
         is Animal.Dog -> animal.feedDog()
-        // Branch with both primary and guard conditions. Returns `feedCat()` when `Animal` is `Cat` and is not `mouseHunter`
+        // Branch with both primary and guard conditions. Calls `feedCat()` when `animal` is `Cat` and is not `mouseHunter`
         is Animal.Cat if !animal.mouseHunter -> animal.feedCat()
-        // Returns "Unknown animal" if none of the above conditions match
+        // Prints "Unknown animal" if none of the above conditions match
```

Comment:
JFYI
I think it should be `Calls` and `Prints` instead of `Returns` here
just like in `control-flow.md`